ngraph/frontend/paddlepaddle/src/op/pow.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "pow.hpp"
+#include <ngraph/builder/make_constant.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <paddlepaddle_frontend/utility.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs pow(const NodeContext& node)
+                {
+                    auto x = node.get_ng_input("X");
+                    auto dtype = x.get_element_type();
+                    Output<Node> factor_node;
+                    if (node.has_ng_input("FactorTensor"))
+                    {
+                        factor_node = node.get_ng_input("FactorTensor");
+                        if (factor_node.get_element_type() != dtype)
+                            factor_node = std::make_shared<opset6::Convert>(factor_node, dtype);
+                    }
+                    else
+                    {
+                        factor_node = builder::make_constant(
```

Comment:
Please don't use builder::make_constant if you don't need that. It creates Broadcast nodes, use Constant::create instead.